model/src/main/kotlin/OrtResult.kt

```diff
@@ -381,6 +381,13 @@ data class OrtResult(
     @JsonIgnore
     fun getRepositoryConfigResolutions(): Resolutions = repository.config.resolutions.orEmpty()
 
+    /**
+     * Return the [Resolutions] contained in this [OrtResult].
+     */
+    @JsonIgnore
+    fun getResolutions(): Resolutions =
+        getRepositoryConfigResolutions().merge(resolvedConfiguration.resolutions.orEmpty())
```

Comment:
What now confuses me here is: If as discussed [here](https://github.com/oss-review-toolkit/ort/pull/8026/commits/b29ea3287e9d5610ac8019ac9d525fb1d35d8448#r1425568974) resolutions from the repository config are already part of the resolved configuration, why do they need to be merged here?

Reply:
I decided for this, as not all stages have the resolutions in the resolved configuration, e.g. like the analyzer.

Reply:
> IMO if resolutions have not been resolved, it's fine to ignore those from the repository configuration. 

I've changed it accordingly.